### DISCUSSION TITLE
docs/

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#823] [Announcements]
# docs/

这是海外技术资料平台的文档主页。 为满足海外项目生产和科研需要，按照港航所的统一部署，海外专职团队于2020年开始专业资料库建设。资料库涉及到海外业务的方方面面的报告、规范、标准、基础数据、书籍等数据成果。直至目前，已经积累了不少资料并相应的开展了一些资料库的基础建设工作，包括：国际标准规范库建设、项目资料库建设、技术范本库、纸质书籍收集整理、专业英语...

http://localhost:8080/docs/

**Comment by eli-soft** [comment]
http://localhost:8080/docs/

**Comment by SHIKEAIXY** [comment]
相关问题请前往Gitee.com/SHIKEAIXY/zhenxun/issues/new询问

**Comment by MKDHXY** [comment]
大家要记得打开qqnt的WS反向然后填入地址哦w

**Comment by jackoname** [comment]
ss
